.github/workflows/kubernetes-integration-test.yaml

```diff
@@ -0,0 +1,49 @@
+name: kubernetes-integration-test
+on: [push]
+jobs:
+  kubernetes-integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout code
+        uses: actions/checkout@v2
+      - 
+        name: Go build
+        run: CGO_ENABLED=0 GOOS=linux go build .
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Docker build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          load: true
+          tags: thegreenwebfoundation/grid-intensity-exporter:integration-test
+      - 
+        name: Create kubernetes Kind Cluster
+        uses: helm/kind-action@v1.1.0
+      - 
+        name: Load image into Kind cluster
+        run: kind load docker-image --name chart-testing thegreenwebfoundation/grid-intensity-exporter:integration-test
```

Comment:
This test is similar to the docker test but it loads the image into the kind cluster.